riscv/insns/vfunary0_vv.h

```diff
@@ -0,0 +1,110 @@
+// VFUNARY0 encoding space
+if ((insn.rs1() & 0b11000) == 0b00000) {
```

Comment:
I think this should be split up into 5 files, one per instruction.  I updated `opcodes-rvv` accordingly: https://github.com/riscv/riscv-opcodes/blob/a2bf04f1fa574e5d8045bf557550027e55d14d1a/opcodes-rvv#L324